0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -148,46 +168,51 @@ Content-Type: application/json
 
 ### Payment Setup
```

Comment:
Are there use cases where you might want to send data along with the PSK payment? If so, SPSP should specify a format for the PSK data field.

Reply:
Not quite sure on this one. Supporting something like "source identifier" or "payment memo" might be useful. But there's a lot of use cases where we don't need that, so I think whatever we do I'd like to have the option to leave the application data blank.